node/service/src/lib.rs

```diff
@@ -655,6 +662,10 @@ where
 			relay_chain_interface,
 			relay_chain_slot_duration,
 			import_queue,
+			// TODO: plumb this through CLI properly
+			collator_options: CollatorOptions {
+				relay_chain_rpc_url: Default::default(),
```

Comment:
@notlesh does this mean that we can't really test `--relay-chain-rpc-url` yet ? It matters, because @crystalin  wanted us to start testing this feature

Reply:
I'll follow up with a new PR. It shouldn't be difficult.